FindOpenStudioSDK.cmake

```diff
@@ -19,7 +19,7 @@ if(APPLE)
     set(OPENSTUDIO_EXPECTED_HASH 5ef630a7282b10c8a9bba34f00539a98)
     set(OPENSTUDIO_PLATFORM "Darwin-arm64")
   else()
-    set(OPENSTUDIO_EXPECTED_HASH a5f16ff19c3a36f5a44fd0153da21633)
+    set(OPENSTUDIO_EXPECTED_HASH 5a1e5fdfc61a879a9d72dcf625a83e65)
```

Comment:
@j-lorrey @jmarrec 

This includes hotfix for https://github.com/NREL/OpenStudio/issues/4584, not sure if arm64 on Mac was affected by that issue?

Reply:
arm64 already had libintil8, since I have noticed it was missing during my own testing.

Reply:
Nice thanks!